.travis.yml

```diff
@@ -3,7 +3,8 @@ language: scala
 sudo: false
 
 scala:
-  - "2.11.8"
+  - "2.11.11"
+  - "2.12.2"
```

Comment:
Travis is currently used just for whitesource and publishing, and publishing is done 'in one go' by `+publish`, so the 2 different Scala versions are not needed here. I'll update `CONTRIBUTING.MD` to make this a bit more clear.